airflow/providers/amazon/aws/transfers/redshift_to_s3.py

```diff
@@ -104,28 +107,42 @@ def __init__(  # pylint: disable=too-many-arguments
                 'HEADER',
             ]
 
+    def _build_unload_query(self, credentials, select_query, s3_key, unload_options):
```

Comment:
Please add type hints

Reply:
I'm a bit confused about the type for `credentials`. My debugger shows it's of type `ReadOnlyCredentials` which in turn is a `NamedTuple`. Which type should I put for `credentials`?

Reply:
I would be in favor of `credentials: ReadOnlyCredentials`

Reply:
Added in the last commit.